### ISSUE TITLE
Can't login using Ubuntu 18.04

### ISSUE THREAD

**Issue by Dbevan770** [runelite/runelite#2603]
Hi, I am currently unable to login to the client on my Ubuntu installation as clicking the various login buttons does nothing. Is there a direct link to the login page?

**Comment by deathbeam** [comment]
What login buttons? Loggining to RuneLite or game?

**Comment by deathbeam** [comment]
Message box should pop up if the client fails to open the website. Otherwise it is using your default browser. And the message box will say that it will copy the URL to your clipboard.

**Comment by Dbevan770** [comment]
Nothing appears but I'll try pasting into the browser to see if it copies to the clipboard anyways

**Comment by deathbeam** [comment]
The messagebox should appear, weird. then, I guess it succeds in trying to open your browser but your default browser is set wrong

**Comment by Dbevan770** [comment]
The default browser is Firefox and thats the only one that's installed in Ubuntu.

**Comment by m00fin** [comment]
also having this issue on ubuntu. i click the button and nothing at all happens.

**Comment by jplsek** [comment]
Alright, I found a workaround from [here](https://stackoverflow.com/a/50103533).
TLDR:
```
printf '\xfe\xed\xfe\xed\x00\x00\x00\x02\x00\x00\x00\x00\xe2\x68\x6e\x45\xfb\x43\xdf\xa4\xd9\x92\xdd\x41\xce\xb6\xb2\x1c\x63\x30\xd7\x92' | sudo tee /etc/ssl/certs/java/cacerts
sudo /var/lib/dpkg/info/ca-certificates-java.postinst configure
```
Now you should be able to to log in.

**Comment by m00fin** [comment]
the workaround posted works for me! great work :D thanks a lot

**Comment by aedelgod** [comment]
Can also confirm this issue on Ubuntu 18.04 LTS btw with the latest jar file.

**Comment by bugii** [comment]
on oracle java this did the trick for me:
sudo apt install ca-certificates-java

**Comment by jshade2710** [comment]
to further comment on @bugii, I had to add hardware acceleration in a .sh file to get GE/highscores lookup. My .sh file has: '#!/bin/sh
java -jar ~/Desktop/RuneLite.jar --mode=OFF/

**Comment by deathbeam** [comment]
This looks like problem on users side with not having certs properly configured, the guide is now on wiki and not much we can we do about that, closing.

**Comment by fuckgithub03** [comment]
Have the same issue, fixing certificates does not remedy the problem. Login link opens a module briefly then does nothing.
